app/views/proposals/_proposal_form.html.haml

```diff
@@ -4,6 +4,10 @@
 
     = f.input :subtitle, as: :string
 
+    = f.input :speakers, as: :select,
+      collection: options_for_select(@users.map {|user| ["#{user.name}", user.id]}, @event.speakers.pluck(:id)),
```

Comment:
Perhaps it makes sense to also show the email of the user next to the name (helps differentiate between duplicate accounts with the same name)